packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## NEXT
+## 2.10.0
 
-* Updates minimum supported SDK version to Flutter 3.19/Dart 3.3.
+* Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+* Fix platform detection so this package correctly reports compatibility with
```

Comment:
A changelog for a package doesn't need to say that the changes are about "this package". How about just "Fixes detection of WebAssembly compatibility."?

Reply:
done